// out: SyntaxError: unexpected "fn"
while (true) fn foo() {}
